pkg/azure/client.go

```diff
@@ -124,6 +156,41 @@ func (az *azClient) getGroup() (r.Group, error) {
 	return az.groupsClient.Get(az.ctx, string(az.resourceGroupName))
 }
 
+func (az *azClient) getVnet(resourceGroupName ResourceGroup, vnetName ResourceName) (n.VirtualNetwork, error) {
+	return az.virtualNetworksClient.Get(az.ctx, string(resourceGroupName), string(vnetName), "")
+}
+
+func (az *azClient) findSubnet(vnet n.VirtualNetwork, subnetPrefix string) (subnet n.Subnet, err error) {
+	for _, subnet := range *vnet.Subnets {
+		if *subnet.AddressPrefix == subnetPrefix {
+			return subnet, nil
+		}
+	}
+	err = errors.New("Unable to find subnet")
```

Comment:
Where are we using `err`?

It'd be nice to declare `errors.New("Unable to find subnet")` once in the global scope and use it here.

Reply:
will move this out